### PR TITLE
README.md: minor improvements

### PR DESCRIPTION
**Pull request by maxim-belkin**
Mostly stylistic changes. Feel free to reject.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,50 +1,60 @@
 # Linuxbrew Dockerfiles
 
-Dockerfiles for using Linuxbrew on various distributions of Linux.
+Linuxbrew Dockerfiles for various distributions of Linux.
```
Comment (sjackman) [inline README.md]: I prefer the previous wording for this line.

**Review by sjackman — CHANGES_REQUESTED** [review]
One change, otherwise looks good. Thanks, Maxim!

File: README.md (hunk 1 above)
Comment (maxim-belkin) [reply README.md]: Previous wording is not accurate, is it? `...for using Linuxbrew on various distributions of Linux` conveys the idea that these Dockerfiles allow one to use Linuxbrew on various Linux distributions. These files, however, allow one to use Linuxbrew on any OS with Docker installed.
`Linuxbrew Dockerfiles for various distributions of Linux.` is not super accurate either...

Comment (sjackman) [reply README.md]: How about…
```suggestion
Dockerfiles for installing Linuxbrew on various distributions of Linux.
```
To me the purpose of the Dockerfile is to install Linuxbrew on an a distribution of Linux inside Docker. It's understood that Docker is running on some other arbitrary distribution of Linux.

Comment (maxim-belkin) [reply README.md]: Let me, actually, undo this change. I don't agree with "on various distributions of Linux." so it'll be easier to just not touch it in this PR.

**Pull request merged**